scripts/aggregate_cells.py

```diff
@@ -74,19 +113,21 @@ def aggregate(
     )
 
     # counting cells in each well and saving it as csv file
-    print("Counting cells within each well")
+    logging.info("Counting cells within each well")
     cell_count_df = single_cell_profile.count_cells()
 
-    print("Saving cell counts in: {}".format(cell_count_out))
+    logging.info(f"Saving cell counts in: {cell_count_out}")
     cell_count_df = cell_count_df.merge(
         platemap_df, left_on="Image_Metadata_Well", right_on="well_position"
     ).drop(["WellRow", "WellCol", "well_position"], axis="columns")
     cell_count_df.to_csv(cell_count_out, sep="\t", index=False)
 
-    print("aggregating cells")
+    # aggregating singel cells into aggregate profile
+    logging.info("Aggregating cells")
     single_cell_profile.aggregate_profiles(
         output_file=aggregate_file_out, compression_options="gzip"
     )
+    logging.info(f"Aggregate profile saved in : {aggregate_file_out}")
```

Comment:
for these logging statements, I'm wondering if it is worth capturing plate names here as well. Or maybe these details are captured elsewhere and doing so here would be redundant?

Reply:
I am assuming that the `plate_name` that you are suggesting is the name associated with the bar code, correct? if that is the case, the logging system does capture the plate name! [souce](https://github.com/WayScience/CytoPipe/pull/14/files#diff-ad800222cde8f8e6f52b6ea3d9cc4c5e51b240355424face998be371d5ce3dc7R86-R93)